packages/sn-filter-pane/src/components/ListboxContainer.tsx

```diff
@@ -22,6 +22,8 @@ const ListboxContainer = ({
     options,
   } = store.getState();
 
+  const optionsString = JSON.stringify(options);
```

Comment:
Is this necessary in order for the useEffect below to trigger?
This is something we would like to avoid actually.

Does it behave the same if you get the options directly from the `useOptions` hook instead?

Reply:
Not necessary to stringify the object actually. I can still use the options object itself to trigger useEffect.

Reply:
If uncertain, a solution for now could be to just add: `options?.direction`